src/models/itemCard.ts

```diff
@@ -0,0 +1,17 @@
+import { imageURL } from "src/utils/utils";
+
+export class ItemCardModel {
+  title: string;
+  uuid: string;
+  url: string;
+  imageID: string;
+  imageURL: string;
+
+  constructor(data: any) {
+    this.uuid = data.uuid;
+    this.title = data.title;
+    this.url = data.href;
+    this.imageID = data.imageID;
```

Comment:
I'm a bit confused. The issue with images not loading was because `CollectionCardModel` uses `data.image_id` but this is `data.imageID`. Where are these two being set and why can't this one be `data.image_id` to be consistent? (Or it can be `imageID`, as long as it's the same for consistency)

Reply:
this is being set by the response so this model is really just guessing at what that field will be named and had a mismatch with the mock data that I made up. it's imageID [here](https://docs.google.com/document/d/1ace9GWSZ3LgbytiPoLal0zauaU6mqycpjqz6RAI0y9k/edit#heading=h.wtj1goozv89y) but image_id in old DC. I'll make it consistently imageID for all collections/items for now but in general, I think technically these field names are out of scope until we actually connect the endpoints cc @avertrees